plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -873,8 +883,17 @@ public void testDecimalBackedByINT64()
         for (int precision = 4; precision <= MAX_PRECISION_INT64; precision++) {
             int scale = ThreadLocalRandom.current().nextInt(precision);
             MessageType parquetSchema = parseMessageType(format("message hive_decimal { optional INT64 test (DECIMAL(%d, %d)); }", precision, scale));
-            ContiguousSet<Long> longValues = longsBetween(-1_000, 1_000);
             int expectedPrecision = precision;
+
+            ImmutableList.Builder<Long> writeValues = ImmutableList.builder();
```

Comment:
This code snipped appears to be repeating (with slight modifications).
Consider creating a method for code reuse.